ocp_install_env.sh

```diff
@@ -130,6 +130,48 @@ EOF
   fi
 }
 
+function renderVIPs() {
+    # This function helps to write in a YAML format multiple resources.
+    # Example: apiVIPs and ingressVIPs
+    resource="${2}";
+
+    echo "    ${1}"
+    for data in ${resource//,/ }; do
+        echo "    - ${data}"
+    done
+}
+
+function setVIPs() {
+    # apiVIP and ingressVIP both has been DEPRECATED in 4.12 in favor of apiVIPs and ingressVIPs.
+    # This functions helps to write the new apiVIPs/ingressVIPs format or set the old fields.
+    # Args: resource name, example: apiVIPs or ingressVIPs
+    case "${1}" in
+    "apivips")
+        if printf '4.12\n%s\n' "$(openshift_version)" | sort -V -C; then \
+            # OCP version is equals or newer as 4.12 and supports the new VIPs fields
+            renderVIPs "apiVIPs:" "${API_VIP}"
+        else
+            # OCP version is older as 4.12 and does not support the new VIPs fields
+cat <<EOF
+    apiVIP: ${API_VIP}
+EOF
```

Comment:
nit: as you need `echo` in renderVIPs too, you could use `echo "    apiVIP: ${API_VIP}"` here to... Probably makes it clearer to read here.